user_api.py

```
@@ -0,0 +1,52 @@
+from flask import Blueprint, request, jsonify
+import sqlite3
+
+user_api = Blueprint('user_api', __name__)
+
+def get_db():
+    return sqlite3.connect('ecom.db')
+
+@user_api.route('/user/create', methods=['POST'])
+def create_user():
+    data = request.json
+    name = data['name'] if 'name' in data else None
```

Ruff (0.12.2)

Use `data.get('name', None)` instead of an `if` block

Replace with `data.get('name', None)`

(SIM401)